llama.h

```diff
@@ -230,6 +230,8 @@ extern "C" {
                              const char * path_model,
             struct llama_context_params   params);
 
+    LLAMA_API void llama_split_layers_weighted(struct llama_context * ctx, std::vector<float> device_weights);
```

Comment:
We should not use C++ in the C-style API

Reply:
Sounds good, would replacing with a float array and a size_t length be sufficient?

Reply:
Should be fixed now